engine/dlib/src/test/test_httpclient.cpp

```diff
@@ -788,9 +788,9 @@ TEST_P(dmHttpClientTest, Post)
         m_ToPost = "";
 
         for (int j = 0; j < n; ++j) {
-            char buf[2] = { (char)((rand() % 255) - 128), 0 };
-            m_ToPost.append(buf);
+            int8_t buf[2] = { (int8_t)((rand() % 255) - 128), 0 };
```

Comment:
Annoying bug: it didn't wrap as one was used to. it stayed in 0-255 range.
I suspect some deprecated behavior was finally caught up by c++17 (and we haven't ran this test on such a compiler before?)